pkg/options/types.go

```diff
@@ -145,7 +145,7 @@ func (n *NodeType) GetNodeFieldSelector() string {
 		klog.InfoS("Using node type is nil")
 		return EmptyFieldSelector()
 	}
-	pattern := "[^a-zA-Z0-9_,-]+"
+	pattern := "[^a-zA-Z0-9_,-\\.]+"
```

Comment:
I would add some tests that verify that different hostnames/fqdns work.

Reply:
Removed this regexp.